src/analysis/log_tonality.lean

```diff
@@ -0,0 +1,93 @@
+/-
+Copyright (c) 2018 Bolton Bailey. All rights reserved.
```

Comment:
Unless I'm wrong, the moved (hence earliest) lemma comes from #10985, which was in 2021.
```suggestion
Copyright (c) 2021 Bolton Bailey. All rights reserved.
```

Reply:
Sorry, my bad from copying the template